.env.example

```
@@ -4,6 +4,9 @@ APP_KEY=
 APP_DEBUG=true
 APP_URL=http://localhost
 
+LEARNING_URL=
+LEARNING_CLIENT_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LEARNING_CLIENT_SECRET key should go before the LEARNING_URL key

(UnorderedKey)